pkg/plugins/policies/core/rules/rules.go

```diff
@@ -290,6 +360,24 @@ func (r *Rule) GetBackendRefOrigin(hash common_api.MatchesHash) (core_model.Reso
 
 type Rules []*Rule
 
+func (rs Rules) NewCompute(element Element) *Rule {
```

Comment:
Using New is odd if nothing is created in the method

Reply:
yeah, I see.
This is just for testing currently, I'll rename it to `Compute` once the PR is ready